src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -170,7 +170,7 @@ partial class ApplicationEngine
             [OpCode.MIN] = 200,
             [OpCode.MAX] = 200,
             [OpCode.WITHIN] = 200,
-            [OpCode.PACK] = 7000,
+            [OpCode.PACK] = 15000,
             [OpCode.UNPACK] = 7000,
```

Comment:
Why they are different?

Reply:
Have fixed, they should be equal if counted time exhausted in func `CheckZeroReferred`